data/20-gnome-initial-setup.rules

```diff
@@ -21,8 +22,16 @@ polkit.addRule(function(action, subject) {
                          action.id.indexOf('org.freedesktop.realmd.') === 0 ||
                          action.id.indexOf('org.freedesktop.RealtimeKit1.') === 0);
 
+    var setupPrograms = [
+        '/usr/lib/gnome-initial-setup/eos-setup-live-user',
+        '/usr/libexec/eos-setup-demo-user',
```

Comment:
That's a good point. To be honest, we can probably revert this change if we just use policies for everything, and I believe those policies use `@libexecdir@` already.